Kitodo/src/main/java/org/kitodo/production/helper/HelperComparator.java

```diff
@@ -48,6 +52,52 @@ public void setSortType(SortType sortType) {
         this.sortType = sortType;
     }
 
+    /**
+     * Compares two ordered lists and returns the differences.
+     *
+     * @param left
+     *            the one list
+     * @param right
+     *            the other list
+     * @return a map with the differences, mapped as T on boolean, where
+     *         {@code true} if it appears in the first list and not in the
+     *         second, {@code false} if it appears in the second list and not in
+     *         the first.
+     */
+    public static Map<Integer, Boolean> compareLists(TreeSet<Integer> left, TreeSet<Integer> right) {
```

Comment:
It should be in different class, maybe there where it is used or in ProcessListComparator.

HelperComparator is old utility class which compares legacy stuff. It is going to be removed.